### DISCUSSION TITLE
Date Formatting Issue in Lookup: Unable to Achieve Expected Format (DD-MM-YYYY)

### DISCUSSION THREAD

**Discussion by madnol** [mia-platform/community#365] [Q&A]
### Which section?

Console

### Which is the version?

back-kit@1.4.7, microlc-backend@0.8.1, microlc-frontend@0.9.0

### Description

I am trying to format a date in a lookup, I have tried using formatDate and rawObject, but the returned result is always lacking the expected format DD-MM-YYYY

```
"customerId": {
          "type": "string",
          "format": "lookup",
          "excludeFromSearch": false,
          "label": {
            "it": "Paziente",
            "en": "Patient"
          },
          "lookupOptions": {
            "lookupValue": "authUserId",
            "lookupFields": [
              "firstName",
              "lastName",
              "birthDate"
            ],
            "lookupDelimiter": " ",
            "lookupDataSource": "users"
          },
          "formOptions": {
            "readOnly": true
          },
          "visualizationOptions": {
            "template": "{{{firstName}}} {{{lastName}}} - {{{formatDate \"DD-MM-YYYY\" birthDate}}}"
          }
        },
```

### Actual Outcome

1996-05-30T22:00:00.000Z

### Expected Outcome

30-05-1996

**Comment by clabene** [comment]
Hi @madnol,
thanks for reaching out.

Unfortunately, this is level of customization is not supported as of now.
We would recommend approaching this task using [MongoDB Views](https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/mongo_views), or [Writable Views](https://docs.mia-platform.eu/docs/runtime_suite/crud-service/writable_views).
